tests/main/interfaces-timezone-control/task.yaml

```diff
@@ -6,7 +6,14 @@ details: |
 
 prepare: |
     # Install a snap declaring a plug on timezone-control
-    "$TESTSTOOLS"/snaps-state install-local test-snapd-timedate-control-consumer
+    if systemctl list-unit-files | grep systemd-timesyncd | awk '{print $2}' | MATCH "enabled"; then
```

Comment:
Why not `systemctl is-enabled systemd-timesyncd`?

Reply:
heh, good point, copied it over from the timeserver-control test, might as well fix it in both